blockio-api/src-linux/System/FS/BlockIO/Internal.hs

```diff
@@ -1,30 +1,49 @@
-{-# LANGUAGE CPP #-}
+{-# LANGUAGE CPP       #-}
+{-# LANGUAGE DataKinds #-}
```

Comment:
What is this used for?

Reply:
Probably a remnant of some version of the code I had before this. I'll remove it!